_pytest/assertion/util.py

```diff
@@ -171,10 +171,22 @@ def _diff_text(left, right, verbose=False):
     """
     from difflib import ndiff
     explanation = []
+
+    def to_unicode_text(binary_text):
```

Comment:
i believe we should try to make this function name a bit more speaking

Reply:
That's the best I could come up with by reading the git blame message for the line in question... do you have a suggestion?

If we don't get a better name I think that's not a big deal TBH given that it is a local function, but definitely if we can get an improved name the better.

Reply:
`escape_for_readable_diff` perhaps - its a bit tricky to name this one indeed

Reply:
I like it, updated!